### PR TITLE
docs: Update _index.md addressing #11575

### LINKED ISSUE #11575: Docs feedback: /docs/sources/send-data/fluentbit/_index.md
Under the [Kubernetes](https://grafana.com/docs/loki/latest/send-data/fluentbit/#kubernetes) section the grafana fluent-bit helm chart is referenced, but it has been deprecated. I would suggest adding instruction/references to the official fluent bit helm installation.

### PR DESCRIPTION
**Pull request by PeterStolz**
**What this PR does / why we need it**:
It improves the docs by addressing #11575 allowing users to use the official fluent-bit helm chart over the deprecated grafana one

**Which issue(s) this PR fixes**:
Fixes #11575 

**Special notes for your reviewer**:
@JStickler wanted to take a look at this.

**Checklist**
- [ x ] Reviewed the [`CONTRIBUTING.md`](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) guide (**required**)
- [ x ] Documentation added
- [ ] Tests updated
- [ ] `CHANGELOG.md` updated
  - [ ] If the change is worth mentioning in the release notes, add `add-to-release-notes` label
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/setup/upgrade/_index.md`
- [ ] For Helm chart changes bump the Helm chart version in `production/helm/loki/Chart.yaml` and update `production/helm/loki/CHANGELOG.md` and `production/helm/loki/README.md`. [Example PR](https://github.com/grafana/loki/commit/d10549e3ece02120974929894ee333d07755d213)
- [ ] If the change is deprecating or removing a configuration option, update the `deprecated-config.yaml` and `deleted-config.yaml` files respectively in the `tools/deprecated-config-checker` directory. [Example PR](https://github.com/grafana/loki/pull/10840/commits/0d4416a4b03739583349934b96f272fb4f685d15)

### CODE REVIEW

File: docs/sources/send-data/fluentbit/_index.md [hunk 1]
```diff
@@ -80,6 +80,8 @@ helm install fluent-bit fluent/fluent-bit -f values.yaml
 
 By default it will collect all containers logs and extract labels from Kubernetes API (`container_name`, `namespace`, etc..).
 
+If you also want to host your Loki insance inside the cluster install the [official Loki helm chart](https://grafana.com/docs/loki/latest/setup/install/helm/).
```
Comment (MichelHollands) [inline docs/sources/send-data/fluentbit/_index.md:83]:
```suggestion
If you also want to host your Loki instance inside the cluster install the [official Loki helm chart](https://grafana.com/docs/loki/latest/setup/install/helm/).
```

File: docs/sources/send-data/fluentbit/_index.md [hunk 2]
```diff
@@ -8,7 +8,9 @@ weight:  500
 ---
 # Fluent Bit client
 
-[Fluent Bit](https://fluentbit.io/) is a fast and lightweight logs and metrics processor and forwarder that can be configured with the [Grafana Loki output plugin](https://docs.fluentbit.io/manual/pipeline/outputs/loki) to ship logs to Loki. You can define which log files you want to collect using the [`Tail`](https://docs.fluentbit.io/manual/pipeline/inputs/tail) or [`Stdin`](https://docs.fluentbit.io/manual/pipeline/inputs/standard-input) data pipeline input. Additionally, Fluent Bit supports multiple `Filter` and `Parser` plugins (`Kubernetes`, `JSON`, etc.) to structure and alter log lines.
+[Fluent Bit](https://fluentbit.io/) is a fast and lightweight logs and metrics processor and forwarder that can be configured with the Grafana Fluent Bit Plugin described here or with the [Fluent-bit Loki output plugin](https://docs.fluentbit.io/manual/pipeline/outputs/loki) to ship logs to Loki. 
+This plugin has more configuration options compared to the built in fluent bit loki plugin.
```
Comment (JStickler) [inline docs/sources/send-data/fluentbit/_index.md:12]:
```suggestion
This plugin has more configuration options compared to the built-in Fluent Bit Loki plugin.
```

File: docs/sources/send-data/fluentbit/_index.md [hunk 3]
```diff
@@ -26,23 +28,59 @@ docker run -v /var/log:/var/log \
...
+        Mem_Buf_Limit 5MB
+        Skip_Long_Lines On
+
+  outputs: |
+    [Output]
+        Name grafana-loki
+        Match kube.*
+        Url ${FLUENT_LOKI_URL}
+        Labels {job="fluent-bit"}
+        LabelKeys level,app # this sets the values for actual loki streams and the other labels are converted to structured_metadata https://grafana.com/docs/loki/latest/get-started/labels/structured-metadata/
```
Comment (JStickler) [inline docs/sources/send-data/fluentbit/_index.md:67]:
```suggestion
        LabelKeys level,app # this sets the values for actual Loki streams and the other labels are converted to structured_metadata https://grafana.com/docs/loki/latest/get-started/labels/structured-metadata/
```

**Review by JStickler — APPROVED** [review]
[Docs team]  LGTM, thanks for the contribution!

**Pull request merged**